test/unit/helper/StateWidgetTest.php

```diff
@@ -46,10 +46,14 @@ class StateWidgetTest extends TestCase
     private const TRANSITION_ID_1 = 'id_1';
     private const TRANSITION_ID_2 = 'id_2';
 
-    /** @var JsonWorkflow|MockObject  */
+    /**
+     * @var JsonWorkflow|MockObject
+     */
     private $workflowModelMock;
 
-    /** @var ResourceWorkflowService|MockObject */
+    /**
+     * @var ResourceWorkflowService|MockObject
```

Comment:
The previous annotation for attributes is actually more widely used now, so we should not fix these cases.

Reply:
fixed! thanks! :)